src/main/webapp/common.js

```diff
@@ -238,6 +238,47 @@ async function displayDeliverySlots() {
   });
 }
 
+/**
+ * Fetches user's delivery slots from Java servlet and displayes them on deliverySlots.html page.
+ */
+async function displayJourneys() {
+  let idToken = await getIdToken();
```

Comment:
Looks like `idToken` is never reassigned?
```suggestion
  const idToken = await getIdToken();
```

Reply:
Yes, I changed it to const.